### PR TITLE
Consider sent back as closed in workload report

### PR DESCRIPTION
**Pull request by timpeat**
## Description of change
Consider sent back as closed in workload report

## Link to relevant ticket

[CRIMRE-385](https://dsdmoj.atlassian.net/browse/CRIMRE-385)

## Notes for reviewer

A typo, was causing sent back applications not be counted as closed in the workload report.

## How to manually test the feature

[CRIMRE-385]: https://dsdmoj.atlassian.net/browse/CRIMRE-385?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by zheileman — APPROVED** [review]
👍

File: app/read_models/received_on_reports/configuration.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Configuration
     ].freeze
 
     CLOSING_EVENTS = [
-      Reviewing::SendBack, Reviewing::Completed
+      Reviewing::SentBack, Reviewing::Completed
```
Comment (arthurashman) [inline app/read_models/received_on_reports/configuration.rb:8]: 😮

Comment (timpeat) [reply app/read_models/received_on_reports/configuration.rb:8]: I know 😞

Comment (arthurashman) [reply app/read_models/received_on_reports/configuration.rb:8]: Easily done. And we all missed it in peer review too!

**Pull request merged**